### ISSUE TITLE
Lose audio when use --merge-output-format

### ISSUE THREAD

**Issue by LeXuanKhanh** [ytdl-org/youtube-dl#29350]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.06.06**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
[youtube] iq7npmYlOK8: Downloading webpage
[download] Destination: Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.f248.webm
[download] 100% of 18.09MiB in 00:02
[download] Destination: Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.f251.webm
[download] 100% of 3.48MiB in 00:00
[ffmpeg] Merging formats into "Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.mp4"
Deleting original file Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.f248.webm (pass -k to keep)
Deleting original file Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.f251.webm (pass -k to keep)
```

## Description

Hi I try using the --merge-output-format on Windows to force the default extension of the video file to be mp4 file, but the output video always end up losing the audio, i'm using youtube-dl with ffmpeg
My command: 
```
./youtube-dl  -f '(bestvideo+bestaudio/best)' --merge-output-format mp4 https://www.youtube.com/watch?v=iq7npmYlOK8
```

**Comment by october262** [comment]
retry with youtube-dl -f mp4 https://www.youtube.com/watch?v=iq7npmYlOK8

**Comment by kikuyan** [comment]
If you run `ffmpeg -i "Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.mp4"`, won't you see the lines like below? (you don't seem to have ffprobe, so ignore the error 'At least one output file must be specified')

>  Stream #0:0(eng): Video: vp9 (Profile 0) (vp09 / 0x39307076), yuv420p(tv, bt709), 1920x1080, 688 kb/s, SAR 1:1 DAR 16:9, 30 fps, 30 tbr, 16k tbn, 16k tbc (default)
>  Stream #0:1(eng): Audio: opus (Opus / 0x7375704F), 48000 Hz, stereo, fltp, 129 kb/s (default)

I've run youtube-dl with your options, and mpv plays this mp4 fine.

**Comment by LeXuanKhanh** [comment]
@kikuyan, yes I have that line when I run your command line, my full output is below 
```
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'D:\desktop\flutter_youtube_downloader_1.2.0\Anne-Marie - Alarm (Marshmello Remix)-iq7npmYlOK8.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2mp41
    encoder         : Lavf58.76.100
  Duration: 00:03:40.02, start: 0.000000, bitrate: 825 kb/s
  Stream #0:0(eng): Video: vp9 (Profile 0) (vp09 / 0x39307076), yuv420p(tv, bt709), 1920x1080, 688 kb/s, SAR 1:1 DAR 16:9, 30 fps, 30 tbr, 16k tbn, 16k tbc (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1(eng): Audio: opus (Opus / 0x7375704F), 48000 Hz, stereo, fltp, 129 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]
```

**Comment by LeXuanKhanh** [comment]
> retry with youtube-dl -f mp4 https://www.youtube.com/watch?v=iq7npmYlOK8

Hi @october262, your option only get the non-audio mp4 file, beside it choose some random quality ( in my case it not my desire quality which is 1080)

**Comment by kikuyan** [comment]
Audio isn't lost, is it?

**Comment by LeXuanKhanh** [comment]
> Audio isn't lost, is it?
no, audio is lost in that file which I run the command
I have fixed it by using `--recode-video`  or specify the m4a audio format when choosing the audio

**Comment by kikuyan** [comment]
Audio isn't lost. In your ffmpeg output:
>   Stream #0:1(eng): Audio: opus (Opus / 0x7375704F), 48000 Hz, stereo, fltp, 129 kb/s (default)
